src/transforms/group.js

```diff
@@ -266,6 +270,15 @@ const reduceFirst = {
   }
 };
 
+const reduceTitle = {
+  reduce(I, X) {
+    const groups = sort(rollup(I, V => V.length, i => X[i]), second);
+    const top = groups.slice(-5).reverse();
+    if (top.length < groups.length) top.push(["Other", sum(groups.slice(0, -5), second)]);
```

Comment:
```suggestion
    const N = 5;
    const groups = sort(rollup(I, V => V.length, i => X[i]), second);
    const top = groups.slice(-5).reverse();
    if (top.length < groups.length) top.push([groups.length === N + 1 ? groups[N][0] : "Other", sum(groups.slice(0, -N), second)]);
```

Reply:
(if there are exactly 6 groups we can use the 6th's name.)

Reply:
Changed to show the number of other categories, as well as the number of constituent elements.